src/jdk.incubator.vector/share/classes/jdk/incubator/vector/Byte128Vector.java

```diff
@@ -375,21 +375,28 @@ public final Byte128Mask test(Test op) {
     @Override
     @ForceInline
     public final Byte128Mask compare(Comparison op, Vector<Byte> v) {
-        return super.compareTemplate(Byte128Mask.class, op, v);  // specialize
+        return super.compareTemplate(Byte128Mask.class, op, v, null);  // specialize
     }
 
     @Override
     @ForceInline
     public final Byte128Mask compare(Comparison op, byte s) {
-        return super.compareTemplate(Byte128Mask.class, op, s);  // specialize
+        return super.compareTemplate(Byte128Mask.class, op, broadcast(s), null);  // specialize
     }
 
     @Override
     @ForceInline
     public final Byte128Mask compare(Comparison op, long s) {
-        return super.compareTemplate(Byte128Mask.class, op, s);  // specialize
+        return super.compareTemplate(Byte128Mask.class, op, broadcast(s), null);  // specialize
     }
 
+    @Override
+    @ForceInline
+    public final Byte128Mask compare(Comparison op, Vector<Byte> v, VectorMask<Byte> m) {
+        return compareTemplate(Byte128Mask.class, op, v, ((Byte128Mask)m));
```

Comment:
One more space between `(Byte128Mask)m`, and no brackets is needed for it after then.